### PR TITLE
Alibaba.adrive version 2.1.9.693

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20200)

### CODE REVIEW

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.installer.yaml:5]: image

```suggestion
PackageVersion: 2.1.9

```

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://yunpan.aliyun.com/downloads/apps/desktop/aDrive.exe?spm=aliyundrive.index.0.0.2d836020dLdn8F&file=aDrive.exe
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.installer.yaml:9]:
```suggestion
  InstallerUrl: https://yunpan.aliyun.com/downloads/apps/desktop/aDrive.exe
```

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 2.1.9
```

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
+PackageLocale: en-US
+Publisher: 阿里云盘
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml:7]: image

```suggestion
Publisher: Alibaba Group
```

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
+PackageLocale: en-US
+Publisher: 阿里云盘
+PackageName: aDrive
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.locale.en-US.yaml:8]: image

```suggestion
PackageName: 阿里云盘
```

File: manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Alibaba.adrive
+PackageVersion: 2.1.9.693
```
Comment (ImJoakim) [inline manifests/a/Alibaba/adrive/2.1.9.693/Alibaba.adrive.yaml:5]: image

```suggestion
PackageVersion: 2.1.9
```

**Comment by ImJoakim** [comment]
Please see my comments above 😃
You also need to change the folder name.

- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666175932
- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666176119
- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666176441
- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666176950
- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666177587
- https://github.com/microsoft/winget-pkgs/pull/20200#discussion_r666177801

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/20200#pullrequestreview-702039331

**Comment by ImJoakim** [comment]
@oxygen-dioxide It's failing because the file has a capital A in alibaba. Rename the following files:

`Alibaba.adrive.installer.yaml` -> `alibaba.adrive.installer.yaml`
`Alibaba.adrive.locale.en-US.yaml` -> `alibaba.adrive.locale.en-US.yaml `
`Alibaba.adrive.yaml`  -> `alibaba.adrive.yaml `

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**